doc/rbd/rbd-snapshot.rst

```diff
@@ -4,22 +4,24 @@
 
 .. index:: Ceph Block Device; snapshots
 
-A snapshot is a read-only copy of the state of an image at a particular point in
-time. One of the advanced features of Ceph block devices is that you can create
-snapshots of the images to retain a history of an image's state. Ceph also
-supports snapshot layering, which allows you to clone images (e.g., a VM image)
-quickly and easily. Ceph supports block device snapshots using the ``rbd`` 
-command and many higher level interfaces, including `QEMU`_, `libvirt`_, 
-`OpenStack`_ and `CloudStack`_.
+A snapshot is a read-only logical copy of an image at a particular point in
+time, aka a checkpoint. One of the advanced features of Ceph block devices is
+that you can create snapshots of images to retain point-in-time state history.
+Ceph also supports snapshot layering, which allows you to clone images (e.g., a
+VM image) quickly and easily. Ceph block device snapshots are managed using the
+``rbd``  command and multiple higher level interfaces, including `QEMU`_,
+`libvirt`_, `OpenStack`_ and `CloudStack`_.
 
 .. important:: To use RBD snapshots, you must have a running Ceph cluster.
 
-.. note:: Because RBD does not know about the file system, snapshots are
-	  `crash-consistent` if they are not coordinated with the mounting
-	  computer. So, we recommend you stop `I/O` before taking a snapshot of
-	  an image. If the image contains a file system, the file system must be
-	  in a consistent state before taking a snapshot or you may have to run
-	  `fsck`. To stop `I/O` you can use `fsfreeze` command. See
+.. note:: Because RBD does not know about any filesystem within an image
+          (volume), snapshots are not `crash-consistent` unless they are
+          coordinated within the mounting (attaching) operating system.
+          So, we recommend you pause or stop I/O before taking a snapshot.
```

Comment:
s/So, we recommend/We therefore recommend that/

Reply:
ack